src/App.tsx

```diff
@@ -7,14 +10,21 @@ import { ImageComponent } from './components/image/Image.component.tsx';
 import './App.module.css';
 
 function App() {
```

Comment:
![Screenshot 2024-05-08 at 18 44 04](https://github.com/3apy6a/fe-react-2024/assets/115080771/0e5661b2-3f32-4114-b557-be6c1951b1ae)

![Screenshot 2024-05-08 at 18 44 10](https://github.com/3apy6a/fe-react-2024/assets/115080771/85828f46-bb1e-4be5-883e-3950227bd5c8)

I see that your footer in one line with  content, maybe better do the same with header

Reply:
done